### ISSUE TITLE
Stop workout without advise

### ISSUE THREAD

**Issue by leogbar** [blockbasti/just_another_workout_timer#77]
Hello, when I have in a workout on static bike for example, and switch to another app, like WhatsApp, stop the workout without advise, think may be for a conflict with audio, this not happens with another type of apps, simpletask for example.

**Comment by blockbasti** [comment]
Hey, thanks for the bug report. Can you give some information about the device you're using? Because normally, the app has no problem running in the background.

**Comment by leogbar** [comment]
Yes, its a Xiaomi Mi A2 Lite with Android One, with the last updates, and mainly use apps open source, and (more or less) inevitably  closed sourced from Google, WhatsApp, etc.

**Comment by leogbar** [comment]
And is Android 10
